content/en/docs/Installation/azure.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Installation on Microsoft Azure"
+linkTitle: "Installation on Azure"
+weight: 3
+date: 2025-03-24
+description: Install Kairos on Microsoft Azure
+---
+
+This page describes how to install Kairos on [Microsoft Azure](https://azure.microsoft.com/) after you have created a disk image. Since release v3.3.2, Kairos pipeline is pushing a public OS image to Azure which you can use.
+If you want to build a custom image, you can follow the instructions in the [Build Kairos appliances]({{< relref "../advanced/build" >}}) page.
+
+## Prerequisites
+
+- An Azure account with permissions to create VMs.
+- An Azure compatible image of Kairos. You can use the public image provided by Kairos (see below) or [build your own image]({{< ref "/docs/Reference/auroraboot.md#generate-raw-disk-images" >}}) (for Azure, that would be the `.vhd` format) and upload it to your Azure resource group ([check how the Kairos CI does it](https://github.com/kairos-io/kairos/blob/cbc6e033cda624e61b2050439b1a95c04fbe78de/.github/workflows/upload-cloud-images.yaml#L158-L241)).
+
+## Deploy a VM
+
+
+Visit the [Community Images page](https://portal.azure.com/#view/Microsoft_Azure_ComputeHub/ComputeHubMenuBlade/~/communityImagesBrowse) in the Azure portal and search for "kairos" in the search box. Multiple results will be returned matching multiple different regions. Click on the result matching the region you intend to use for the Virtual Machine. Find the version you want to use and click on "Create VM".
```

Comment:
do we want to say which publisher the image comes from to avoid bad actors publishing fake Kairos images? Or is that not possible?

Reply:
I don't know. I wanted to do some research on the proper way to do this, that's why I created this separate issue: https://github.com/kairos-io/kairos/issues/3325